### ISSUE TITLE
Skip preserving file extension for generated path

### ISSUE THREAD

**Issue by devarsh** [trayio/babel-plugin-webpack-alias#25]
Babel would emit the files with js extension irrespective of the extension of source files, which would cause problem to test individual files without using webpack or gulp,  since required modules path would never be resolved, becuase generated files would have an extension of `js` and the generated webpack alias path would be referring to `jsx`(i.e original extension)

I've shown an example to support the use case

### .babelrc
```js
{
  "presets": ["es2015", "react","stage-2"],
  "plugins" : [
  ["babel-plugin-webpack-alias", { "config": "./webpack/server/server.js" }]
  ],
}
```

### webpack config
```js
resolve : {
    extensions: ['','.js','.jsx'],
    modules : [srcPath,path.join(basePath,'./node_modules')],
    alias : {
      template: path.join(srcPath,'./template')
    },
  },
```

### es6 code
```js
import Layout from 'template/Layout';
```

### Generating es5 file post babel transformation
```console
src/server/template/Layout.jsx -> lib/server/template/Layout.js
```

### Post babel transformation 
```js
var _Layout = require('./Layout.jsx'); //should be './Layout.js'
var _Layout2 = _interopRequireDefault(_Layout);
```

### Running in node
```console
Error: Cannot find module './Layout.jsx'
```

**Comment by adriantoine** [comment]
What I can do is adding an option `noOutputExtension` which, when set to true:
```json
{
  "presets": ["es2015", "react","stage-2"],
  "plugins" : [
  ["babel-plugin-webpack-alias", { "noOutputExtension": true, "config": "./webpack/server/server.js" }]
  ],
}
```
would output all require statements without extensions:
```js
var _Layout = require('./Layout');
var _Layout2 = _interopRequireDefault(_Layout);
```

Would that solve your issue?

**Comment by devarsh** [comment]
Yes, it would

**Comment by adriantoine** [comment]
Ok, I'm working on it now

**Comment by devarsh** [comment]
Thanks for such a quick response, and the plugin is minimal but very
helpful.

**Comment by adriantoine** [comment]
Thanks! @devarsh

**Comment by adriantoine** [comment]
The change is made, I'll publish a new version

**Comment by adriantoine** [comment]
@devarsh It's just been released in [v2.1.0](https://github.com/trayio/babel-plugin-webpack-alias/releases/tag/v2.1.0)!

**Comment by devarsh** [comment]
@adriantoine thanks alot, i'll check and revert back to you
